Sources/Rendering/OpenGL/glsl/vtkVolumeFS.glsl

```diff
@@ -43,7 +43,7 @@ varying vec3 vertexVCVSOutput;
 //VTK::ImageLabelOutlineOn
 
 #ifdef vtkImageLabelOutlineOn
-uniform int outlineThickness;
+uniform int outlineThickness[256];
```

Comment:
I'm not sure if this is correct or not, but I believe we need to have a fixed size for uniform arrays. I think 256 segments might be enough, but maybe we should target a higher number?

Reply:
Maybe it can be uint8 ? 
1024 should be high enough without taking too much space...